src/ontodia/editor/editLayer.tsx

```diff
@@ -37,9 +38,6 @@ export interface State {
     canDropOnElement?: boolean;
 }
 
-const PLACEHOLDER_ELEMENT_TYPE = '' as ElementTypeIri;
```

Comment:
Do you think it would be possible to use empty `ElementModel.types` array instead of "special" `PLACEHOLDER_ELEMENT_TYPE`?

Reply:
IMHO http://ontodia.org/NewEntity carries more semantics then empty array. Could use our templating if needed as well.